autoload/elm.vim

```diff
@@ -90,287 +90,287 @@ fun! elm#Format()
 	endif
 
-	return fixes
+	return l:fixes
 endf
 
 " Make the given file, or the current file if none is given.
-fun! elm#Make(...)
-	if elm#util#CheckBin("elm-make", "http://elm-lang.org/install") == ""
+function! elm#Make(...) abort
+	if elm#util#CheckBin('elm-make', 'http://elm-lang.org/install') ==# ''
 		return
 	endif
 
-	call elm#util#Echo("elm-make:", "building...")
+	call elm#util#Echo('elm-make:', 'building...')
 
-	let input = (a:0 == 0) ? expand("%:p") : a:1
-	let fixes = elm#Build(input, g:elm_make_output_file, g:elm_make_show_warnings)
+	let l:input = (a:0 == 0) ? expand('%:p') : a:1
+	let l:fixes = elm#Build(l:input, g:elm_make_output_file, g:elm_make_show_warnings)
 
-	if len(fixes) > 0
-		call elm#util#EchoWarning("", "found " . len(fixes) . " errors")
+	if len(l:fixes) > 0
+		call elm#util#EchoWarning('', 'found ' . len(l:fixes) . ' errors')
 
-		call setqflist(fixes, 'r')
+		call setqflist(l:fixes, 'r')
 		cwindow
 
-		if get(g:, "elm_jump_to_error", 1)
+		if get(g:, 'elm_jump_to_error', 1)
 			ll 1
 		endif
 	else
-		call elm#util#EchoSuccess("", "Sucessfully compiled")
+		call elm#util#EchoSuccess('', 'Sucessfully compiled')
 
 		call setqflist([])
 		cwindow
 	endif
 endf
 
 " Show the detail of the current error in the quickfix window.
-fun! elm#ErrorDetail()
-	if !empty(filter(tabpagebuflist(), 'getbufvar(v:val, "&buftype") ==# "quickfix"'))
-		exec ":copen"
-		let linenr = line(".")
-		exec ":wincmd p"
+function! elm#ErrorDetail() abort
+	if !empty(filter(tabpagebuflist(), 'getbufvar(v:val, "&buftype") ==? "quickfix"'))
+		exec ':copen'
+		let l:linenr = line('.')
+		exec ':wincmd p'
 		if len(s:errors) > 0
-			let detail = s:errors[linenr-1].details
-			if detail == ""
-				let detail = s:errors[linenr-1].overview
+			let l:detail = s:errors[l:linenr-1].details
+			if l:detail ==# ''
+				let l:detail = s:errors[l:linenr-1].overview
 			endif
-			echo detail
+			echo l:detail
 		endif
 	endif
 endf
 
 " Open the elm repl in a subprocess.
-fun! elm#Repl()
+function! elm#Repl() abort
 	" check for the elm-repl binary
-	if elm#util#CheckBin("elm-repl", "http://elm-lang.org/install") == ""
+	if elm#util#CheckBin('elm-repl', 'http://elm-lang.org/install') ==# ''
 		return
 	endif
 
 	if has('nvim')
-		term("elm-repl")
+		term('elm-repl')
 	else
 		!elm-repl
 	endif
 endf
 
-function elm#Oracle(filepath, word)
-	let bin = "elm-oracle"
-	let filepath = shellescape(a:filepath)
-	let word = shellescape(a:word)
-	let command = bin . " " . filepath . " " . word
-	return s:ExecuteInRoot(command)
+function! elm#Oracle(filepath, word) abort
+	let l:bin = 'elm-oracle'
+	let l:filepath = shellescape(a:filepath)
+	let l:word = shellescape(a:word)
+	let l:command = l:bin . ' ' . l:filepath . ' ' . l:word
+	return s:ExecuteInRoot(l:command)
 endfunction
 
-let s:fullComplete = ""
+let s:fullComplete = ''
 
 " Complete the current token using elm-oracle
-fun! elm#Complete(findstart, base)
+function! elm#Complete(findstart) abort
```

Comment:
I'm sure Vint complains that 'base' is unused, which is true, but the callback function for an omnifunc handler is `function! CompleteFunc(findstart, base)`. Would you mind reverting this line, otherwise vim complains that the function doesn't have the correct number of arguments when I try to autocomplete.